docs/devices/TS011F_plug_1.md

```diff
@@ -27,6 +27,15 @@ pageClass: device-page
 ## Notes
 
 
+### Broken attribute reporting functionality
+
+Starting with firmware version 1.0.5 (which comes pre-flashed on plugs produced since Q4 2021) core functionality on this plug is broken. TuYa has moved attribute reporting of the current plug state (on/off) and energy measurement away from the official Zigbee Cluster Library Specification into a private API. Efforts to implement this private API to restore functionality have not succeeded until now.
```

Comment:
Please clarify that you are only affected if your plug is detected as a TS011F_plug_3